tap_facebook/streams.py

```diff
@@ -42,26 +43,91 @@ def process_account(
             return self.__advance_bookmark(account_id, state, None, tap_stream_id)
 
         prev_bookmark = None
+        fields = fields or [
+            "account_id",
+            "account_name",
+            "account_currency",
+            "ad_id",
+            "ad_name",
+            "adset_id",
+            "adset_name",
+            "campaign_id",
+            "campaign_name",
+            "clicks",
+            "ctr",
+            "date_start",
+            "date_stop",
+            "frequency",
+            "impressions",
+            "reach",
+            "social_spend",
+            "spend",
+            "unique_clicks",
+            "unique_ctr",
+            "unique_link_clicks_ctr",
+            "inline_link_clicks",
+            "unique_inline_link_clicks"
+        ]
         with singer.metrics.record_counter(tap_stream_id) as counter:
+
+            if not start_date:
+                raise ValueError("client: start_date is required")
+
+            since = utils.parse_date(start_date)
+
+            until = date.today() + timedelta(days=-1)
+
+            increment = 1
```

Comment:
This increment is never modified to more than `1`, or at least I don't see where is it modified

Reply:
So the timeranges are always 1 day difference